src/neurotechdevkit/scenarios/_scenario_1.py

```diff
@@ -51,30 +24,28 @@ class _Scenario1Mixin:
         https://asa.scitation.org/doi/pdf/10.1121/10.0013426
     """
 
-    @property
-    def _material_layers(self: _Scenario1MixinProtocol) -> list[tuple[str, Struct]]:
-        return [
-            ("water", materials.water),
-            ("skin", materials.skin),
-            ("cortical bone", materials.cortical_bone),
-            ("trabecular bone", materials.trabecular_bone),
-            ("brain", materials.brain),
-        ]
+    material_layers = [
+        "water",
+        "skin",
+        "cortical_bone",
+        "trabecular_bone",
+        "brain",
+    ]
 
     @property
     def _material_outline_upsample_factor(self) -> int:
         return 8
 
     def _get_material_masks(
-        self: _Scenario1MixinProtocol, problem: stride.Problem
+        self, problem: stride.Problem
     ) -> Mapping[str, npt.NDArray[np.bool_]]:
         return {
             name: _create_scenario_1_mask(name, problem.grid)
-            for name in self.materials.keys()
+            for name in self.material_layers
         }
 
     def _compile_scenario_1_problem(
-        self: _Scenario1MixinProtocol, extent: npt.NDArray[np.float_]
+        self, extent: npt.NDArray[np.float_]
     ) -> stride.Problem:
         # scenario constants
         speed_water = 1500  # m/s
```

Comment:
`c_freq` below is staying constant instead of updating with the user defined value.